packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -459,18 +460,23 @@ export class AzureCommunicationCallAdapter implements CallAdapter {
     });
   }
 
-  public startCall(participants: string[], options?: StartCallOptions): Call | undefined {
+  public startCall(participants: string[] | CommunicationIdentifier[], options?: StartCallOptions): Call | undefined {
```

Comment:
Could this be: `participants: (string | CommunicationIdentifier)[]` or will it always be an array of one _or_ the other

Reply:
Good Point.
We are trying to deprecate the usages of string[] as userID everywhere. Hence, I kept it this way ;)

Reply:
Also `(string | CommunicationIdentifier)[]` would mean that you could mix both types in the same array, which is different from two arrays with that have a different uniform type